code/index.html

```diff
@@ -43,22 +44,24 @@ <h1>Does the person have</h1>
 
       <button id="filter" class="filled-button">FIND OUT</button>
     </aside>
-
+    <!--SECOND SECTION FOR THE BOARD-->
     <section class="board-wrapper">
       <main class="game-board" id="board"></main>
     </section>
-
+    <!--THIRD SECTION-->
     <section id="winOrLose" class="win-or-lose-wrapper">
       <div class="win-or-lose">
         <img
           class="guess-who-icon"
           src="images/guess-who-bubble.png"
-          alt="Guess Who"
-        />
+          alt="Guess Who" />
         <h1 id="winOrLoseText"></h1>
         <button id="playAgain" class="filled-button">PLAY AGAIN</button>
       </div>
     </section>
-    <script src="script.js"></script>
+    <audio id="winnerSound" src="./audios/applause.mp3" preload="auto"></audio>
```

Comment:
Great with some extra sound features. I got a bit scared when the sound of loosing was played :-)